### PR TITLE
Test

### PR DESCRIPTION
**Pull request by linaroos**
## Netlify link
Add your Netlify link here.
PS. Don't forget to add it in your readme as well.

## Collaborators
Add your collaborators here. Write their GitHub usernames in square brackets. If there's more than one, separate them with a comma, like this:
[github-username-1, github-username-2]

### CODE REVIEW

File: README.md
Comment (NellaMori) [inline README.md:1]: don't forget to add the deployed link to the PR description and README file 👍

File: .DS_Store
Comment (NellaMori) [inline .DS_Store:1]: .DS_Store is a file that stores custom attributes of its containing folder, such as folder view options, icon positions, and other visual information, and we don't need to share it in our project. Please add .DS_Store to your .gitignore file (if you don't have one, you need to create it) 👍

**Review by AntonellaMorittu — COMMENTED** [review]
Hi Lina, good job with your chatbot! 🤖 The code is well structured, you meet the requirements and you also put into practice the skills learnt so far, amazing! ⭐  To improve even more the UX you could add validation for the name field so that is not empty for example? I left a few comments in the code, keep up the good work! 👍

**Comment by NellaMori** [comment]
please re-open the pull request to get it approved! :)

**Pull request closed without merging**